sys/include/net/l2util.h

```diff
@@ -133,6 +133,116 @@ int l2util_ipv6_iid_to_addr(int dev_type, const eui64_t *iid, uint8_t *addr);
 int l2util_ndp_addr_len_from_l2ao(int dev_type,
                                   const ndp_opt_t *opt);
 
+/**
+ * @brief Board-specific function to supply a short address to a netdev
+ *
+ * @note  Implement this function in your board code if the board
+ *        provides the means to supply a unique address to a netdev.
+ *
+ * @warning Don't call this function directly, use @ref l2util_generate_short_addr() instead.
+ *
+ * @param[in] netdev    The network device that requested the address
+ * @param[out] addr     The dedicated address for the netdev
+ *
+ */
+size_t short_addr_get_from_board(netdev_t *netdev, uint16_t *addr);
```

Comment:
Name-spacing? How about

```suggestion
size_t l2util_short_addr_from_board(netdev_t *netdev, uint16_t *addr);
```

Also for the other functions.

Reply:
Yea since the actual implementation will be in the respective folder for the individual boards, I thought the `board_*` namespace would be more appropriate.

Reply:
But it doesn't have this either ;-)

Reply:
Changed it back to `board_get_short_addr()`